### PR TITLE
Announcement for Fine Beta 2

### PR DESCRIPTION
**Pull request by JPrause**

### CODE REVIEW

**Comment by JPrause** [comment]
@cybette can you review for Monday's planned merge.
If there are no changes needed, please merge on Monday.

**Comment by JPrause** [comment]
@cybette can you merge this announcement for Fine Beta 2.
QE has passed the smoke tests.

File: site/_posts/2017-04-03-Announcing-Fine-Beta2-Release.html.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: Announcing Fine Beta Release
```
Comment (cybette) [inline site/_posts/2017-04-03-Announcing-Fine-Beta2-Release.html.md]: Better to indicate it's Beta 2 (first Beta announcement has same title)

File: site/_posts/2017-04-03-Announcing-Fine-Beta2-Release.html.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+author: chessbyte
+date: 2017-05-01 07:49:24 UTC
+comments: true
+published: true
+tags: releases announcements
+---
+
+On behalf of the ManageIQ team, we're pleased to announce the Beta 2 release for ManageIQ Fine! You can download the Fine Beta 2 Release [here](http://manageiq.org/download/). You'll find it under the "Pre-release (Fine-1-Beta2)" section.
```
Comment (cybette) [inline site/_posts/2017-04-03-Announcing-Fine-Beta2-Release.html.md]: > You'll find it under the "Pre-release (Fine-1-Beta2)" section.

A bit redundant, and also the sections on the download page change with each release. I'd leave this line out but it's up to you.

**Comment by cybette** [comment]
@JPrause couple minor stuff, I'll merge right after you've taken care of them. Thanks!

**Pull request merged**